src/os/bluestore/BlockDevice.h

```diff
@@ -77,6 +79,14 @@ struct IOContext {
       --num_running;
     }
   }
+
+  void set_return_value(int r) {
+    r = r;
```

Comment:
i think the param needs to be named something else where

Reply:
yeah, it's rather confusing... 😓